package.json

```diff
@@ -186,6 +186,6 @@
     "redux-saga": "^1.1.3",
     "reselect": "^4.1.5",
     "scroll-into-view-if-needed": "^2.2.16",
-    "yup": "^0.29.0"
+    "yup": "^0.30.0"
```

Comment:
@willm30 I found this version to have a working test context and not cause any issues with existing validation